src/graphql/queries/biggest-winners.js

```diff
@@ -64,7 +64,28 @@ module.exports = async (
     }));
   });
   await Promise.all(promises);
-  winnings = _orderBy(winnings, ['amount'], ['desc']);
```

Comment:
why remove the ordering? youll have to implement `orderBy` param for the query then.

Reply:
the orderBy gives weird order, one result was 7, 200, 0, don't know how.

So I'm just using array.sort.

Reply:
hm oh prob because its sorting by Strings instead of numbers